package.json

```diff
@@ -61,7 +61,8 @@
     "workspaceContains:**/project.clj",
     "workspaceContains:**/shadow-cljs.edn",
     "workspaceContains:**/deps.edn",
-    "onDebugResolve:type"
+    "onDebugResolve:type",
+    "onWebviewPanel:calva.output-view"
```

Comment:
Why do we need this? Not saying we don't, just wondering why =)

Reply:
It's what makes VS Code reload the output view upon VS Code starting up.

Reply:
Cool. TIL!

Reply:
But... I think it doesn't hurt to leave it.